root/report/recordings_with_the_same_name_by_different_artists_with_the_same_name.tt

```diff
@@ -0,0 +1,17 @@
+[%- WRAPPER 'layout.tt' title=l('Recordings with the same name by different artists with the same name') full_width=1 -%]
+
+<h1>[% l('Recordigns with the sae name by different artists with the same name') %]</h1>
+
+<ul>
+    <li>[% l('This report shows all recordings with the same name that have different artist MBIDs with the same name.') %]</li>
+    <li>[% l('These are most likely cases where the artist is incorrect for at least one
+              one of the instances. Make sure to check that the {artist_credit|Artist Credit} is correct as well.',
```

Comment:
I would say: "These are most likely cases where the {ac|artist credit} is incorrect for at least one of the recordings." (Note: no line breaks.)